test/puppetlabs/puppetdb/config_test.clj

```diff
@@ -92,8 +93,7 @@
 (deftest garbage-collection
   (let [config-with (fn [base-config]
                       (-> base-config
-                          (update :database merge
-                                  (first tu/available-postgres-configs))
+                          (update :database merge sample-db-config)
```

Comment:
swapping sample-db-config for available-postgres-configs works here, but available-postgres-configs is used in the sync tests so I don't think we'll be able to drop it as we have